docs/learn/glossary.md

```diff
@@ -1,14 +1,14 @@
 # Glossary
 
-Use this glossary to learn about terms specific to the Terra Protocol.
+This glossary provides definitions of terms specific to the Terra Protocol.
 
 ## Active set
 
 The top 130 [validators](#validator) that participate in consensus, receive rewards, and secure the blockchain.
 
 ## Blockchain
 
-An unchangeable ledger of transactions copied among a network of independent computer systems.
+An immutable ledger of transactions copied among a network of independent computer systems.
```

Comment:
Although "immutable" is a common term among web3 natives, it is not a common word for beginners. "Unchangeable" is more accessible.